### PR TITLE
Accept title attribute for name of input type=image

### PR DESCRIPTION
**Pull request by Jym77**
We were failing [this example](https://act-rules.github.io/rules/59796f#passed-example-3)

### CODE REVIEW

File: packages/alfa-aria/test/name.spec.tsx [hunk 1]
```diff
@@ -1083,6 +1084,20 @@ test(`.from() determines the name of an <input> element with a placeholder
   });
 });
 
+test(`.from() determines the name of an \`<input type="image">\` with a
+      title attribute.`, (t) => {
+  const input = <input type="image" src="foo" title="Search" />;
+
+  // console.dir(Name.from(input, device).toJSON(), { depth: null });
```
Comment (zsofitoth) [inline packages/alfa-aria/test/name.spec.tsx:1091]: Leftover `console.dir` and `reference`

File: packages/alfa-aria/test/name.spec.tsx [hunk 2]
```diff
@@ -1167,7 +1182,7 @@ test(`.from() determines the name of an <input type="image"> element`, (t) => {
   t.deepEqual(Name.from(input, device).toJSON(), {
     type: "some",
     value: {
-      value: "Submit",
+      value: "Submit Query",
```
Comment (zsofitoth) [inline packages/alfa-aria/test/name.spec.tsx:1185]: How come this didn't fail before? 😮

**Review by zsofitoth — APPROVED** [review]
Looks good 👍 Just some leftover console logging :)

File: packages/alfa-aria/test/name.spec.tsx (hunk 2 above)
Comment (Jym77) [reply packages/alfa-aria/test/name.spec.tsx:1183]: I don't know and at this point I'm too afraid to dig 🙈

**Pull request merged**